CDC_Backend/APIs/adminViews.py

```diff
@@ -110,13 +145,25 @@ def updateOfferAccepted(request, id, email, user_type):
     try:
         data = request.data
         offer_accepted = data[OFFER_ACCEPTED]
-        opening = get_object_or_404(Placement, pk=data[OPENING_ID])
+        if  OPENING_TYPE  in data:
+            opening_type= data[OPENING_TYPE]
+        else:
+            opening_type= "Placement"
+        if DEADLINE_DATETIME in data:
```

Comment:
Automatically Creating a deadline when accepting offer if the field is not provided in request .if provided using the given field